### ISSUE TITLE
want to contribute? comment here!

### ISSUE THREAD

**Issue by vilmibm** [vilmibm/tildemush#22]
if you're interested in contributing to this project but aren't sure how, please leave a comment here and we'll work something out :D

**Comment by archangelic** [comment]
💖

**Comment by modgethanc** [comment]
👋

i'm getting a working dev environment set up, and poking through the codebase a bit to familiarize myself with it. i think my best strength might be in fleshing out the UI, whenever you feel ready to have some help with the client code.

**Comment by vilmibm** [comment]
@modgethanc help with the UI is definitely something I would love! I think @selfsame is feeling similarly. There's plenty more work to keep me busy on the server side, so I'd be thrilled if y'all were doing stuff on the client side.

i wonder if the next natural thing is starting on the WITCH editor modal? should we make some specifications for that?

**Comment by modgethanc** [comment]
okay, i'm going to drop a couple of low hanging fruit issues for client side work while i'm getting the hang of poking at urwid!

**Comment by m455** [comment]
I'd definitely been down to write some documentation or a tutorial for beginners for this when it's at a publicly usable state :slightly_smiling_face:

**Comment by benjaminwil** [comment]
I'd be interested in doing some client-side work, but I don't have a lot of Python experience yet and may require hand-holding/a boss once in a while.

**Comment by vilmibm** [comment]
@benjaminwil cool! while #109 was mostly done, would you be interested in adding the further aliases endorphant asked for as well as writing some unit tests for the color code processing stuff?

if that's too much to ask, no worries! if you're interested i can help get the tests stuff started so you can add to it.

**Comment by modgethanc** [comment]
@m455 yooo i think docs/tutorial is super important, your instructions and stuff are always super clear/helpful!! i think this could tie in pretty well with #52 for tutorial room stuff, but also it might be nice to have a non-interactive reference page that should probably all have fairly consistent language and stuff?

**Comment by m455** [comment]
@modgethanc yeah, maybe even a little cheat sheet if more creation commands come into place. just let me know what you want me to write and i'll write it up for you :coffee: :banana:

**Comment by apiarian** [comment]
Hi! I've got a fair bit of python experience (though it's not my primary language anymore) and would love to help in any way I can! Do people usually build and run and test things on their own machines? Anybody working in remote tilde.town sessions?

**Comment by modgethanc** [comment]
I have a dev environment on my local machine, which I find easier for me than a tilde.town setting; it's a little finicky to set up the database, but the instructions in the README worked well for me!
-------- Original Message --------

**Comment by miker2049** [comment]
✨ bump✨
